jmh/src/main/scala/filodb.jmh/IntSumReadBenchmark.scala

```diff
@@ -31,7 +31,7 @@ class IntSumReadBenchmark {
   import IntSumReadBenchmark._
   val NumRows = 10000
 
-  val chunkSet = ChunkSet(dataset, partKey, rowIt.map(TupleRowReader).take(NumRows).toSeq, TestData.nativeMem)
+  val chunkSet = ChunkSet(dataset, partKey, 0, rowIt.map(TupleRowReader).take(NumRows).toSeq, TestData.nativeMem)
```

Comment:
Suggestion: default the ingestionTime to 0 so that we can avoid changing the call in tests.  Will make merging easier for other PRs.

Reply:
A default of 0 for testing is dangerous. It makes it too easy to forget about it in code that really needs to specify it.